plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/api/product-attributes.php

```diff
@@ -30,6 +30,14 @@ public function setUp(): void {
 		);
 	}
 
+	public static function tearDownAfterClass(): void {
+		parent::tearDownAfterClass();
+		global $wpdb;
+		parent::tearDownAfterClass(); //
```

Comment:
Why do we need to call `parent::tearDownAfterClass()` twice in this function? 👀

Reply:
That's an error :) Thank you for catching it. I'll update.

Reply:
Updated it in https://github.com/woocommerce/woocommerce/pull/32409/commits/8632950ab8a3d6a47d37801ca5aa8ee10330ef4f